WordPress/AbstractThemeSniff.php

```diff
@@ -0,0 +1,1068 @@
+<?php
```

Comment:
No, it doesn't. One is the File doc block, the other the class doc block.

FYI: in the main WPCS a clean up has taken place of all doc block documentation and the TRTCS sniffs should start complying with the latest standards used in WPCS once the latest WPCS has been merged into TRTCS (and a clean up should be done of already merged sniffs).